sycl/test/regression/std_array_vec_constructor.cpp

```diff
@@ -2,15 +2,15 @@
 // the constructor.
 // REQUIRES: windows
 
-// RUN: not %clangxx -O0 -fsycl -D_DEBUG -shared %s &> %t.compile.log
+// RUN: %clangxx -O0 -fsycl -D_DEBUG -shared %s -nostdlib -Xclang --dependent-lib=msvcrtd -fms-runtime-lib=dll_dbg  &> %t.compile.log
```

Comment:
Just remove the log and `FileCheck`.

Reply:
Done in https://github.com/intel/llvm/pull/18213/commits/b0fdd94a698ae0d75e25fc8acf8674ed7d46e5cb